ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.74__ocp-4-11.yaml

```
@@ -0,0 +1,69 @@
+base_images:
+  ubi-minimal:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: apollo-ci
+    namespace: stackrox
+    tag: stackrox-test-0.3.57.1
+releases:
+  latest:
+    release:
+      channel: fast
+      version: "4.11"
+resources:
+  '*':
+    requests:
+      cpu: 2000m
+      memory: 4000Mi
+test_binary_build_commands: .openshift-ci/dispatch.sh test-binary-build-commands
+tests:
+- as: qa-e2e-tests
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      QA_TEST_TARGET: test
+      TEST_SUITE: openshift-4-qa-e2e-tests
+    test:
+    - ref: stackrox-e2e
+    workflow: ipi-aws
+  timeout: 5h0m0s
+- as: operator-e2e-tests
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      TEST_SUITE: openshift-4-operator-e2e-tests
+    test:
+    - ref: stackrox-e2e
+    workflow: ipi-aws
+  timeout: 5h0m0s
+- as: merge-qa-e2e-tests
+  postsubmit: true
+  steps:
+    cluster_profile: aws
+    env:
+      QA_TEST_TARGET: test
+      TEST_SUITE: openshift-newest-qa-e2e-tests
```

Can be  just
```suggestion
      TEST_SUITE: openshift-qa-e2e-tests
```

but won't block a PR on that.